modules/preview-react/side-panel/lib/NewSidePanel/SidePanelToggleButton.tsx

```diff
@@ -0,0 +1,70 @@
+/** @jsx jsx */
+import * as React from 'react';
+import {css, jsx} from '@emotion/core';
+import {space} from '@workday/canvas-kit-react/tokens';
+import {createComponent, useModelContext} from '@workday/canvas-kit-react/common';
+import {IconButton, IconButtonProps} from '@workday/canvas-kit-react/button';
+import {transformationImportIcon} from '@workday/canvas-system-icons-web';
+
+import {SidePanelModel, SidePanelModelContext, useSidePanelToggleButton} from './hooks';
+
+export interface SidePanelToggleButtonProps
+  extends Omit<IconButtonProps, 'aria-label'>,
+    React.ButtonHTMLAttributes<HTMLButtonElement> {
+  'aria-label'?: string;
```

Comment:
I understand this is necessary as well as the `Omit` above because `IconButton` requires an `aria-label`. I think we have an issue that removes this requirement because there are plenty of examples where an `aria-label` is not required.

Reply:
https://github.com/Workday/canvas-kit/issues/894

Reply:
Yeah, `aria-label` needs to be optional.